src/administration/web/configure-project.md

```diff
@@ -37,10 +34,37 @@ If you check the `Super user` box, this user will be an administrator of the pro
 
 The `Domains` screen allows you to manage your domains that your project will be accessible at.
 
-![Manage domains of your Platform.sh project.](/images/ui-conf-project-domains.png)
+![project domain](/images/settings_domain.png)
 
 More information on how to [setup your domain](/golive/steps.md).
 
 > **note**
 >
 > Platform.sh expects an ASCII representation of the domain here. In case you want to use an internationalized domain name you can use the [conversion tool provided by Verisign](http://mct.verisign-grs.com/) to convert your IDN domain to ASCII.
+
+
+## Certificates
+
+The `Certificates` screen allows you to manage your project's TLS certificates that enable HTTPS. 
+
+![project certs](/images/settings_certificates.png)
+
+You can view current certificates by hovering over one on the list and clicking the `View` link that appears, or you can add a new certificate by clicking the `Add` button a the top of the page. 
+
+![add certs](/images/add_cert.png)
+
+
+## Deploy Key
+
+The `Deploy Key` page provides the SSH key that Platform.sh will use when trying to access external private Git repository during the build process.
+
+This is useful if you want to reuse some code components across multiple projects and manage those components as dependencies of your project.
+
+![project deploy key](/images/settings_Deploy Key.png)
+
+## Variables
+
+The `Variables` screen allows you to define the variables that will be available project-wide - that is, in each environment.
```

Comment:
Add: "It also allows you define variables that will be available during the build process."